specification/sql/resource-manager/Microsoft.Sql/preview/2019-06-01-preview/FailoverManagedInstance.json

```diff
@@ -0,0 +1,177 @@
+            "in": "path",
+            "description": "The name of the managed instance to failover.",
+            "required": true,
+            "type": "string"
+          },
+          {
+            "name": "replicaType",
+            "in": "query",
+            "description": "The type of replica to be failed over.",
+            "required": false,
+            "type": "string",
+            "enum": [
+              "Primary",
+              "ReadableSecondary"
+            ],
+            "x-ms-enum": {
+              "name": "ReplicaType",
+              "modelAsString": true
+            }
+          },
+          {
+            "$ref": "#/parameters/SubscriptionIdParameter"
+          },
+          {
+            "$ref": "#/parameters/ApiVersionParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "Successfully completed managed instance failover."
+          },
+          "202": {
+            "description": "Managed instance failover is in progress."
+          },
+          "default": {
+            "description": "*** Error Responses: ***\n\n * 400 ManagementServiceFeatureDisabled - User attempted to use a feature which is disabled.\n\n * 400 ManagedInstanceFailoverThrottled - There was a recent failover on the managed instance.\n\n * 400 ManagedInstanceFailoverNotSupportedOnSKU - This type of customer initiated failover is not supported on the given SKU.\n\n * 404 SubscriptionDoesNotHaveServer - The requested server was not found\n\n * 404 ServerNotInSubscriptionResourceGroup - Specified server does not exist in the specified resource group and subscription.\n\n * 409 ManagedInstanceNotInStateToFailover - The managed instance is currently in a state such that failover cannot be issued."
+          }
+        },
+        "x-ms-long-running-operation": true,
+        "x-ms-examples": {
+          "Failover a managed instance.": {
+            "$ref": "./examples/FailoverManagedInstance.json"
+          }
+        }
+      }
+    }
+  },
+  "definitions": {},
+  "parameters": {
+    "SubscriptionIdParameter": {
+      "name": "subscriptionId",
+      "in": "path",
+      "description": "The subscription ID that identifies an Azure subscription.",
+      "required": true,
+      "type": "string",
+      "x-ms-parameter-location": "client"
+    },
+    "ApiVersionParameter": {
+      "name": "api-version",
+      "in": "query",
+      "description": "The API version to use for the request.",
+      "required": true,
+      "type": "string",
+      "x-ms-parameter-location": "client"
+    },
+    "ResourceGroupParameter": {
+      "name": "resourceGroupName",
+      "in": "path",
+      "description": "The name of the resource group that contains the resource. You can obtain this value from the Azure Resource Manager API or the portal.",
+      "required": true,
+      "type": "string",
+      "x-ms-parameter-location": "method"
+    },
+    "ServerNameParameter": {
```

Comment:
There are many unused and duplicate definitions here. They should be removed. I looked over other json in SQL, seems there are copy and paste since they exhibit same pattern. This creates problems in our codegen(autorest) and producing duplicate class schemas. 

I opened an [issue](https://github.com/Azure/azure-rest-api-specs/issues/9640) on our side to work with you to clean up these swaggers in the future.